layouts/layout-basics/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "new",
```

Comment:
You can probably change this to reflect the name of the course!

`Awesome Learning - CSS Layout`